Presentation.Web/Controllers/API/BaseApiController.cs

```diff
@@ -170,6 +178,8 @@ protected User KitosUser
             }
         }
 
+        protected int UserId => Convert.ToInt32(User.Identity.Name);
```

Comment:
Det skurer lidt med et "Name" og ToInt32 .. men sådan er det måske bare?

Reply:
Der sådan ASP net authentication sætter den på identity. Det er ikke specielt fedt, men det er i det mindste ensartet for hhv. bearer token auth og cookie auth